content/blog/azure-deployment-environments/index.md

```diff
@@ -0,0 +1,91 @@
+        kind: Storage
+        resourceGroupName: ${resource-group-name}
+        sku:
+            name: Standard_LRS
+    ```
+
+This is all you need to define a Pulumi environment in ADE. If you prefer to use a programming language like C#, TypeScript, Python, or Go, you will also need to add the code next to the `Pulumi.yaml` file.
+
+Once the environment definition is added to the catalog, a user simply chooses an applicable environment in their developer portal. After accepting parameter values, ADE will run the Pulumi program in the container to provision Azure resources. In the case of the above program, it will create a new Storage Account in the environment, and the user will be able to use the resources once they are provisioned by Pulumi.
+
+## Getting Started
+
+To get started, you must have the **Azure Deployment Environments** service configured. You can manually create one in the Azure Portal by following the [instructions](https://learn.microsoft.com/en-us/azure/deployment-environments/quickstart-create-and-configure-devcenter) or using our [sample Pulumi program](https://github.com/pulumi/azure-deployment-environments/tree/main/Provisioning/ade) to provision all the required resources.
+
+Once you have an ADE instance, you can start authoring your **environment definitions**. You can use the [Pulumi Azure Deployment Environments samples](https://github.com/pulumi/azure-deployment-environments/tree/main/Environments) as a starting point. The folder contains four simple Pulumi programs in four different languages, each creating a Storage Account in the environment. The programs also show how to use parameters to integrate with ADE’s configuration system.
+
+Once you have the definitions, you will **publish them to a GitHub or Azure DevOps repository** and [**attach the repository as a Catalog in Azure Deployment Environments**](https://learn.microsoft.com/en-us/azure/deployment-environments/how-to-configure-catalog?tabs=DevOpsRepoMSI). You can do so manually using the Azure Portal or configure it within your Pulumi project, as illustrated in [our sample](https://github.com/pulumi/azure-deployment-environments/blob/1a9633cd31977be2b78cd727b21225ff7b48913d/Provisioning/ade/index.ts#L132-L141).
+
+Finally, **navigate to the [Microsoft developer portal](https://devportal.microsoft.com)** to create a new environment. You will see the list of available templates, including the ones you just published.
+
+## Bring a Custom Container Image
+
+The Pulumi container image [`pulumi/azure-deployment-environments`](https://hub.docker.com/repository/docker/pulumi/azure-deployment-environments) is a great starting point for your environment definitions. However, if you want to customize the behavior to your needs, you can also bring your own container image to run Pulumi programs in ADE. To do so, refer to the [Configure a container image to execute deployments with Pulumi](https://aka.ms/ade/pulumi-docs) article on Microsoft Learn.
```

Comment:
Link is not working.

Reply:
Yes, they will publish later today, same as us